errata_tool/product_version.py

```diff
@@ -14,6 +14,21 @@ def __init__(self, id_or_name, data=None):
         self.data = data
         self.url = self._url + '/product_versions/%s' % self.id_or_name
 
+    def releasedBuild(self, package_name):
+        """Get the released build for this package of this Product Version.
+
```

Comment:
this should have a `:param` for the `package_name` variable.

So AIUI for the example given, you would have passed `ProductVersion('rhel-7').releasedBuild('ceph')`

Reply:
Sure, will add it.